src/Sentry.Unity/SystemInfoAdapter.cs

```diff
@@ -31,6 +31,8 @@ internal interface ISentrySystemInfo
         bool? SupportsComputeShaders { get; }
         bool? SupportsGeometryShaders { get; }
         int? GraphicsShaderLevel { get; }
+        Lazy<bool>? IsDebugBuild { get; }
+        public string? InstallMode { get; }
```

Comment:
```suggestion
       string? InstallMode { get; }
```

Reply:
access modifier in interface? I thought that wouldn't compile

Reply:
Yeah, I'll remove it. For the info, starting from C#8 you're allowed to use `private` or `public` modifiers in `interface`s.